web/src/components/tasks/NaicsCodeTask.test.tsx

```diff
@@ -329,6 +341,33 @@ describe("<NaicsCodeTask />", () => {
       fireEvent.click(screen.getByText(Config.taskDefaults.removeText));
       expect(currentUserData().taskProgress[taskId]).toEqual("IN_PROGRESS");
     });
+
+    it("doesn't display the edit or remove button when the naics code is present and the operating phase is up and running", () => {
+      renderPage("UP_AND_RUNNING");
+      expect(screen.queryByText(Config.taskDefaults.editText)).not.toBeInTheDocument();
+      expect(screen.queryByText(Config.taskDefaults.removeText)).not.toBeInTheDocument();
+      expect(screen.getByTestId("naics-code-tooltip")).toBeInTheDocument();
+    });
+
+    it("displays the edit or remove button when the naics code is present and the operating phase is not up and running", () => {
+      renderPage("FORMED_AND_REGISTERED");
+      expect(screen.getByText(Config.taskDefaults.editText)).toBeInTheDocument();
+      expect(screen.getByText(Config.taskDefaults.removeText)).toBeInTheDocument();
+    });
+
+    it("displays the tooltip when the naics code is present and the operating phase is up and running", () => {
+      renderPage("UP_AND_RUNNING");
+      expect(screen.queryByText(Config.taskDefaults.editText)).not.toBeInTheDocument();
+      expect(screen.queryByText(Config.taskDefaults.removeText)).not.toBeInTheDocument();
+      expect(screen.getByTestId("naics-code-tooltip")).toBeInTheDocument();
+    });
+
+    it("doesn't display the tooltip when the naics code is present and the operating phase is not up and running", () => {
```

Comment:
might be easier to read all these tests if nested under a `describe("when the naics code is present")`